htdocs/themes/xbootstrap5/css/dark-mode.css

```
@@ -0,0 +1,269 @@
+html[data-bs-theme="dark"] .xoopsCode {
+    background-color: #0f172a;
+    color: #e5e7eb;
+    border-color: rgba(255, 255, 255, .16);
+}
+
+html[data-theme="dark"] .breadcrumb,
+html[data-theme="dark"] .pagination,
+html[data-bs-theme="dark"] .breadcrumb,
+html[data-bs-theme="dark"] .pagination {
+    --bs-breadcrumb-divider-color: var(--bs-secondary-color, #aab4c0);
+    color: var(--bs-body-color, #e5e7eb);
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)